actionmailer/lib/action_mailer/delivery_job.rb

```diff
@@ -12,9 +12,15 @@ class DeliveryJob < ActiveJob::Base # :nodoc:
 
     rescue_from StandardError, with: :handle_exception_with_mailer_class
 
-    def perform(mailer, mail_method, delivery_method, params, *args) #:nodoc:
-      mailer_class = params ? mailer.constantize.with(params) : mailer.constantize
-      mailer_class.public_send(mail_method, *args).send(delivery_method)
+    before_perform do
+      ActiveSupport::Deprecation.warn <<~MSG.squish
+        Sending mail with DeliveryJob is deprecated and
+        will be removed in Rails 6.1. Please use NewDeliveryJob instead.
```

Comment:
This should say `MailDeliveryJob` now. Also let's add back in the Paramertized deprecation since I was wrong about when it was added.